src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -320,6 +324,27 @@ public B getOwnBuilding()
         return getOwnBuilding(getExpectedBuildingClass());
     }
 
+    @NotNull
+    public IColony getColony()
+    {
+        return getOwnBuilding().getColony();
+    }
+
+    @NotNull
+    public ServerLevel getServerLevel() {
```

Comment:
Formatting. Could make this only one getLevel which returns ServerLevel then, this never runs on the client side